config/mekanism/BoxBlacklist.txt

```diff
@@ -11,10 +11,16 @@ astralsorcery:blockstarlightinfuser 0
 astralsorcery:blockwell 0
 astralsorcery:blockattunementrelay 0
 astralsorcery:blockritualpedestal 0
-astralsorcery:blockaltar
-astralsorcery:blockcelestialcrystals
+astralsorcery:blockaltar 0
```

Comment:
Not appending a number in the BoxBlacklist.txt means it's a fuzzy match, all meta values. Please undo this change.

Reply:
> it's a fuzzy match, all meta values.

No, its not. 
First, lines without number at the end listed in debug.log file with error spelling "Cant parse line" or something like this.
Second, on current E2E version you still can put box on Altairs or Celestia Crystals, that obiviously shows that blacklist isn't working.